### PR TITLE
Contributing guide + improved devex + direct users to the ask forum

### PR DESCRIPTION
**Pull request by Conner1115**
Waiting on https://github.com/replit/replit-docs-2/pull/265

### CODE REVIEW

**Review by lunaroyster — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -82,21 +82,5 @@ function App() {
...
-3. Once built, you can publish the package with `npm publish`.  Make sure you increment the version.
-4. Update the changelog when publishing.
-5. Copy the URL from the webview and install it as an extension in your Repl.
-
-### Local Development
-1. Clone this repository with `git clone https://github.com/replit/extensions`.
-2. Navigate into the folder with `cd extensions`.
-3. Run `npm run dev` to build the package and run the development server.
-4. Expose localhost to an ngrok link with `ngrok http <port>`.
-5. Copy the https ngrok link and install that as an extension (note: some of the filesystem APIs won't behave correctly on localhost)
```
Comment (lunaroyster) [inline README.md:99]: i think we should add some of this to a CONTRIBUTING file in this repo

**Comment by Conner1115** [comment]
I added a `contributing.md` file and also improved the devex for developing with this on Replit.

**Comment by Conner1115** [comment]
btw what's `is-prod-frozen` doing in this PR?

File: README.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ The Replit Extensions client is a module that allows you to easily interact with
  - [Documentation](https://docs.replit.com/extensions)
   - [Resources](https://docs.replit.com/extensions/resources)
   - [API Modules](https://docs.replit.com/extensions/category/api-reference)
-  - [React Hooks](https://docs.replit.com/extensions/category/react-hooks)
- - [Discord Server](https://discord.gg/replit)
+  - [React Client](https://docs.replit.com/extensions/category/react)
+ - [Discourse Category](https://ask.replit.com/c/extensions)
```
Comment (masad-frost) [inline README.md:15]:
```suggestion
 - [Ask Forum](https://ask.replit.com/c/extensions)
```

**Review by masad-frost — CHANGES_REQUESTED** [review]
Let's also remove the installation and usage section tbh and point people to the docs, instead.

File: .replit [hunk 3]
```diff
@@ -13,6 +13,11 @@ XDG_CONFIG_HOME = "/home/runner/.config"
 PATH = "/home/runner/$REPL_SLUG/.config/npm/node_global/bin:/home/runner/$REPL_SLUG/node_modules/.bin"
 npm_config_prefix = "/home/runner/$REPL_SLUG/.config/npm/node_global"
 
+[extension]
+isExtension = true
+buildCommand = "cd example && npm run build"
+outputDirectory = "./example/dist"
```
Comment (lunaroyster) [inline .replit:19]: oh neat, i forgot about this

**Comment by lunaroyster** [comment]
yeah agree, the links will probably rot

**Comment by Conner1115** [comment]
Are we good here?

**Pull request merged**